clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml

```diff
@@ -0,0 +1,114 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: pj-rehearse-plugin
+    namespace: ci
+  spec:
+    selector:
+      app: prow
+      component: pj-rehearse-plugin
+    ports:
+    - port: 80
+      targetPort: 8888
+    type: ClusterIP
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: pj-rehearse-plugin
+    labels:
+      app: prow
+      component: pj-rehearse-plugin
+    annotations:
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"pj-rehearse:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"pj-rehearse-plugin\")].image"}]'
+    namespace: ci
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    selector:
+      matchLabels:
+        app: prow
+        component: pj-rehearse-plugin
+    template:
+      metadata:
+        labels:
+          app: prow
+          component: pj-rehearse-plugin
+      spec:
+        containers:
+        - name: pj-rehearse-plugin
+          image: registry.ci.openshift.org/ci/pj-rehearse:latest
+          args:
+          - --dry-run=false
+          - --server=true
+          - --log-level=debug
+          - --pre-check=false
+          - --comment-on-pr-creation=false
+          - --prowjob-kubeconfig=/var/kubeconfigs/sa.pj-rehearse.app.ci.config
+          - --kubeconfig-dir=/var/kubeconfigs
+          - --kubeconfig-suffix=config
+          - --normal-limit=10
+          - --more-limit=20
+          - --max-limit=35
+          - --endpoint=/
+          - --hmac-secret-file=/etc/webhook/hmac.yaml
+          - --github-endpoint=http://ghproxy
+          - --github-endpoint=https://api.github.com
+          - --github-token-path=/etc/github/oauth
+          ports:
+            - name: http
+              containerPort: 8888
+          volumeMounts:
+          - mountPath: /var/kubeconfigs
+            name: prowjob-kubeconfig
+          - name: hmac
+            mountPath: /etc/webhook
+            readOnly: true
+          - name: oauth
+            mountPath: /etc/github
+            readOnly: true
+          - name: cache
+            mountPath: /var/tmp
+          resources:
+            requests:
+              memory: "3Gi"
+              cpu: "750m"
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
+            initialDelaySeconds: 20
+            periodSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /healthz/ready
+              port: 8081
+        volumes:
+        - name: prowjob-kubeconfig
+          secret:
+            secretName: pj-rehearse
+        - name: hmac
+          secret:
+            secretName: github-webhook-credentials
+        - name: oauth
+          secret:
+            secretName: github-credentials-openshift-ci-robot
+        - name: cache
```

Comment:
Does the tool generates a cache by default in `/var/tmp`?

Reply:
yeah, that is the default for the git client:
https://github.com/openshift/ci-tools/blob/f80066832fb41b52cb536cf8ec01bbe8e40cee78/vendor/k8s.io/test-infra/prow/git/v2/client_factory.go#L112-L127